templates/taxi/car_detail.html

```diff
@@ -9,4 +9,6 @@ <h1>Drivers</h1>
     <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
   {% endfor %}
   </ul>
```

Comment:
I don't like this page. Make it more beautiful. Make smaller the word `Drivers`, maybe `h3`or `h4` tag would be better. Make text about Manufacturer bigger